app/enterprise/0.34-x/installation/docker.md

```diff
@@ -59,17 +54,17 @@ is required**:
       postgres:9.5
     ```
 
-8. To make the license data easier to handle, export it as a shell variable.
+7. To make the license data easier to handle, export it as a shell variable.
 Please note that **your license contents will differ**! Users with Bintray
-accounts should visit [https://bintray.com/kong/&lt;YOUR_REPO_NAME&gt;/license#files](https://bintray.com/kong/<YOUR_REPO_NAME>/license#files)
+accounts should visit `https://bintray.com/kong/<YOUR_REPO_NAME>/license#files`
```

Comment:
@levfishbluefish this URL doesn't work with using the enterprise repo. Is there another way they can find the license files?

Since it can't be clicked on, and is intended to be copied and altered, I removed the hyperlink.

Reply:
They will receive their license file in an email, but otherwise have to go to bintray. I think changing this to text vs. a clickable link works

Reply:
cool--do you know the right path though? `/license#files` gives me a 404.